### PR TITLE
IDENT-3101 raw key derivation

### PR DESCRIPTION
**Pull request by MaticDiba**

### CODE REVIEW

File: aries_cloudagent/askar/tests/test_store.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from aries_cloudagent.core.error import ProfileError
+from asynctest import TestCase as AsyncTestCase
+
+from ..store import AskarStoreConfig
+
+class TestStoreConfig(AsyncTestCase):
+    async def test_init_success(self):
+        config = {
+            "key_derivation_method": "RAW",
```
Comment (vidp1) [inline aries_cloudagent/askar/tests/test_store.py:9]: "Raw" would be better, to also test the .lower() part.

File: aries_cloudagent/askar/tests/test_store.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    async def test_init_should_fail_when_key_missing(self):
+        config = {
+            "key_derivation_method": "RAW",
+            "storage_type": "postgres"
+        }
+
+        with self.assertRaises(ProfileError):
+            askar_store = AskarStoreConfig(
+                config
+            )
```
Comment (vidp1) [inline aries_cloudagent/askar/tests/test_store.py:31]: New line, not sure if it's needed, but github is not glad. :)

File: aries_cloudagent/config/tests/test_wallet.py [hunk 3]
```diff
@@ -238,3 +238,29 @@ async def test_wallet_config_seed_no_public_did(self):
...
+        self.injector.bind_instance(BaseWallet, mock_wallet)
+
+        with async_mock.patch.object(
+            MockManager, "provision", async_mock.CoroutineMock()
+        ) as mock_mgr_provision:
+            mock_mgr_provision.return_value = self.profile
+
+            await test_module.wallet_config(self.context, provision=True)
+
+            mock_mgr_provision.assert_called_once_with(self.context, {'key_derivation_method': 'derivation_method'})
```
Comment (vidp1) [inline aries_cloudagent/config/tests/test_wallet.py:266]: New line?

File: aries_cloudagent/askar/tests/test_store.py (hunk 2 above)
Comment (acuderman) [reply aries_cloudagent/askar/tests/test_store.py:31]: Yes, It is required. Linter also complains :)

File: aries_cloudagent/config/tests/test_wallet.py (hunk 3 above)
Comment (MaticDiba) [reply aries_cloudagent/config/tests/test_wallet.py]: added

File: aries_cloudagent/askar/tests/test_store.py (hunk 2 above)
Comment (MaticDiba) [reply aries_cloudagent/askar/tests/test_store.py]: added

File: aries_cloudagent/askar/tests/test_store.py (hunk 1 above)
Comment (MaticDiba) [reply aries_cloudagent/askar/tests/test_store.py]: changed and moved to variable

**Review by acuderman — APPROVED** [review]

**Review by vidp1 — APPROVED** [review]

**Pull request merged**